src/allotropy/allotrope/schema_parser/update_units.py

```diff
@@ -47,7 +44,7 @@ class {unit_name}(HasUnit):
     unit: str = {UNIT_STRING_OVERRIDES.get(unit, unit_str)}"""
             )
 
-    with open(CUSTOM_MODELS_PATH, "w") as f:
```

Comment:
just checking, are `CUSTOM_MODELS_PATH` and `UNITS_SCHEMAS_PATH` still used now, with this change? (should we remove?)

Reply:
Hmmm good point, they are technically still used in `GENERATED_SHARED_PATHS`, which is backed up and then passed to the unit update code, but no reason to have separate constants. Fixed.